syntax = "proto3";

package gbf;

import "domain.proto";

message RaidBossesResponse {

  repeated RaidBoss raidBosses = 1;
};